source/style-guide/includes/terms/d.rst

```diff
@@ -31,6 +31,14 @@ D
    database
      Use *database* as one word.
 
+   date
```

Comment:
[Suggestion]

At the top of the Alphabetical List of Terms page, we have a table with icons for special types of terms. I think we should use the "Avoid" icon here like we do for "access (v)".